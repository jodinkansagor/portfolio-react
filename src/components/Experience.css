.experienceName {

  h2 {
    text-align: center;
  }

  dl {
    display: flex;
    flex-direction: row;
    justify-content: space-around;
    width: 100%;
    margin: 0px;
  
    dt {
      text-decoration: underline;
      color: #116466;
      font-size: 20px;
      margin-bottom: 5px;
    }
  
    dd {
      font-size: 18px;
      padding-left: 10px;
      margin: 0px;
    }  
  }

  
  .other {
    margin-top: 5px;
  }
}


