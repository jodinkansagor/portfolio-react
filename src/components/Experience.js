import React from 'react';
import styles from '../components/Experience.css';

const Experience = () => {
  return (
    <section className={styles.experienceName}>
      <h2>Experience</h2>
      <dl>
        <section className={styles.tech}>
          <dt>Technology</dt>
          <dd>JavaScript</dd>
          <dd>HTML</dd>
          <dd>CSS</dd>
          <dd>React</dd>
          <dd>React hooks</dd>
          <dd>Node.js</dd>
          <dd>Express</dd>
          <dd>PostgrSQL</dd>
          <dd>SQL</dd>
          <dd>MongoDB</dd>
          <dd>Mongoose</dd>
          <dd>Jest</dd>
          <dd>Q-Unit</dd>
        </section>
        <section className={styles.tools}>
          <dt>Tools</dt>
          <dd>Heroku</dd>
          <dd>GitHub</dd>
          <dd>Adobe CS</dd>
          <dd>Adobe xD</dd>
          <dd>Compass</dd>
          <dd>PgAdmin</dd>
          <dd>Compass</dd>
          <dd>Postman</dd>
          <dd className={styles.bottomTool}>Travis</dd>

          <dt className={styles.other}>Other Tools</dt>
          <dd>WordPress</dd>
          <dd>ArcGIS</dd>
          <dd>TDD</dd>
        </section>
      </dl>
    </section>
  );
};

export default Experience;
