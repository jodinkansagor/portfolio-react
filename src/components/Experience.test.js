import React from 'react';
import { shallow } from 'enzyme';
import Experience from './Experience';

describe('Experience component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<Experience />);
    expect(wrapper).toMatchSnapshot();
  });
});