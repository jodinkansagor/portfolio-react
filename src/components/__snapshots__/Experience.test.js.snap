// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Experience component matches a snapshot 1`] = `
<section
  className="experienceName"
>
  <h2>
    Experience
  </h2>
  <dl>
    <section
      className="tech"
    >
      <dt>
        Technology
      </dt>
      <dd>
        JavaScript
      </dd>
      <dd>
        HTML
      </dd>
      <dd>
        CSS
      </dd>
      <dd>
        React
      </dd>
      <dd>
        React hooks
      </dd>
      <dd>
        Node.js
      </dd>
      <dd>
        Express
      </dd>
      <dd>
        PostgrSQL
      </dd>
      <dd>
        SQL
      </dd>
      <dd>
        MongoDB
      </dd>
      <dd>
        Mongoose
      </dd>
      <dd>
        Jest
      </dd>
      <dd>
        Q-Unit
      </dd>
    </section>
    <section
      className="tools"
    >
      <dt>
        Tools
      </dt>
      <dd>
        Heroku
      </dd>
      <dd>
        GitHub
      </dd>
      <dd>
        Adobe CS
      </dd>
      <dd>
        Adobe xD
      </dd>
      <dd>
        Compass
      </dd>
      <dd>
        PgAdmin
      </dd>
      <dd>
        Compass
      </dd>
      <dd>
        Postman
      </dd>
      <dd
        className="bottomTool"
      >
        Travis
      </dd>
      <dt
        className="other"
      >
        Other Tools
      </dt>
      <dd>
        WordPress
      </dd>
      <dd>
        ArcGIS
      </dd>
      <dd>
        TDD
      </dd>
    </section>
  </dl>
</section>
`;
